vcsh.in

```diff
@@ -735,11 +742,13 @@ VCSH_COMMAND=$(echo "$VCSH_COMMAND" | @SED@ 's/-/_/g'); export VCSH_COMMAND
 
 # Source repo-specific configuration file
 # shellcheck source=/dev/null
-[ -r "$XDG_CONFIG_HOME/vcsh/config.d/$VCSH_REPO_NAME" ] \
+# TODO: And we rely on $VCSH_REPO_NAME implictly again
+[ -r "$XDG_CONFIG_HOME/vcsh/config.d/${VCSH_REPO_NAME-}" ] \
```

Comment:
This will only suceed by a somewhat odd side effect, `-r` tests specifically for files not directories so it will fail if the values is unset and the path is only to the directory. While the result is correct, the code logic is pretty opaque.

As the assorted comments here note, the active repo(s) name handling should probably be overhauled entirely.